website/static/css/style.css

```diff
@@ -513,3 +513,13 @@ input[type='file'] {
 .cardbox {
 	width: 20rem;
 }
+
```

Comment:
Please remove this space. Have a look at the entire file, and then add the code in the similar format.
I am pretty sure that there is not new lines between different styles in the file.